src/_includes/tools/dart-compile-js-options.md

```diff
@@ -0,0 +1,113 @@
+#### Options {#prod-compile-options}
```

Comment:
includes are typically not used for sections like this. This can make things harder to maintain (the heading type might change depending on the context where it's included.) IMO it's better to just inline things and use includes for banners and other smaller things.